config/src/main/java/tech/pegasys/pantheon/config/GenesisConfigOptions.java

```diff
@@ -87,4 +89,47 @@ private OptionalLong getOptionalLong(final String key) {
         ? OptionalLong.of(configRoot.getLong(key))
         : OptionalLong.empty();
   }
+
+  public static class Builder {
```

Comment:
nit: This builder could/should also handle the consensus parameters - but given it is currently only used for testing, does not need repairing now.